[2024-09-22T02:44:18.061+0000] {processor.py:186} INFO - Started process (PID=4496) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:44:18.062+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:44:18.064+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:44:18.064+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:44:18.068+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:44:18.067+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:44:18.068+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:44:18.068+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:44:18.068+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:44:18.080+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-09-22T02:45:23.518+0000] {processor.py:186} INFO - Started process (PID=4561) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:45:23.520+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:45:23.522+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:45:23.522+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:45:23.526+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:45:23.525+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:45:23.526+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:45:23.526+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:45:23.526+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:45:23.538+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-09-22T02:46:19.096+0000] {processor.py:186} INFO - Started process (PID=4618) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:46:19.098+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:46:19.100+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:46:19.100+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:46:19.104+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:46:19.103+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:46:19.104+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:46:19.104+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:46:19.105+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:46:19.117+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-09-22T02:47:12.410+0000] {processor.py:186} INFO - Started process (PID=4674) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:47:12.412+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:47:12.414+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:47:12.413+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:47:12.418+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:47:12.417+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:47:12.418+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:47:12.418+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:47:12.419+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:47:12.431+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-09-22T02:48:03.573+0000] {processor.py:186} INFO - Started process (PID=4728) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:48:03.575+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:48:03.577+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:48:03.576+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:48:03.580+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:48:03.580+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:48:03.581+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:48:03.581+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:48:03.581+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:48:03.593+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-09-22T02:48:53.321+0000] {processor.py:186} INFO - Started process (PID=4780) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:48:53.322+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:48:53.325+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:48:53.324+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:48:53.328+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:48:53.327+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:48:53.328+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:48:53.328+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:48:53.329+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:48:53.341+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-09-22T02:49:43.193+0000] {processor.py:186} INFO - Started process (PID=4831) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:49:43.194+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:49:43.197+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:49:43.196+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:49:43.200+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:49:43.199+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:49:43.200+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:49:43.200+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:49:43.201+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:49:43.213+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-09-22T02:50:33.412+0000] {processor.py:186} INFO - Started process (PID=4888) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:50:33.414+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:50:33.416+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:50:33.415+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:50:33.419+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:50:33.419+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:50:33.420+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:50:33.420+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:50:33.420+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:50:33.432+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-09-22T02:51:22.457+0000] {processor.py:186} INFO - Started process (PID=4943) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:51:22.458+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:51:22.461+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:51:22.460+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:51:22.464+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:51:22.463+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:51:22.464+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:51:22.464+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:51:22.465+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:51:22.477+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-09-22T02:52:11.132+0000] {processor.py:186} INFO - Started process (PID=4994) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:52:11.134+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:52:11.136+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:52:11.135+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:52:11.139+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:52:11.139+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:52:11.140+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:52:11.139+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:52:11.140+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:52:11.152+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-09-22T02:53:00.491+0000] {processor.py:186} INFO - Started process (PID=5043) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:53:00.493+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:53:00.495+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:53:00.495+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:53:00.499+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:53:00.498+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:53:00.499+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:53:00.499+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:53:00.499+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:53:00.511+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-09-22T02:53:51.084+0000] {processor.py:186} INFO - Started process (PID=5097) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:53:51.086+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:53:51.088+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:53:51.088+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:53:51.092+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:53:51.091+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:53:51.092+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:53:51.092+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:53:51.093+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:53:51.105+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-09-22T02:54:39.299+0000] {processor.py:186} INFO - Started process (PID=5149) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:54:39.301+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:54:39.303+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:54:39.302+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:54:39.306+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:54:39.306+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:54:39.307+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:54:39.307+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:54:39.307+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:54:39.319+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-09-22T02:55:30.423+0000] {processor.py:186} INFO - Started process (PID=5205) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:55:30.425+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:55:30.427+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:55:30.427+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:55:30.431+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:55:30.430+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:55:30.431+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:55:30.431+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:55:30.431+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:55:30.444+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-09-22T02:56:19.866+0000] {processor.py:186} INFO - Started process (PID=5256) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:56:19.867+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:56:19.869+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:56:19.869+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:56:19.873+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:56:19.872+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:56:19.873+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:56:19.873+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:56:19.874+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:56:19.886+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-09-22T02:57:08.823+0000] {processor.py:186} INFO - Started process (PID=5307) to work on /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:57:08.825+0000] {processor.py:914} INFO - Processing file /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-22T02:57:08.827+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:57:08.826+0000] {dagbag.py:587} INFO - Filling up the DagBag from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:57:08.831+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:57:08.830+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-22T02:57:08.831+0000] {logging_mixin.py:190} INFO - [2024-09-22T02:57:08.831+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-22T02:57:08.831+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-22T02:57:08.844+0000] {processor.py:208} INFO - Processing /home/ubuntu/endtoendairflow_venv/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
